src/overlays/actors/ovl_En_Maruta/z_en_maruta.c

```diff
@@ -287,25 +315,25 @@ void func_80B372CC(EnMaruta* this, PlayState* play) {
     }
 }
 
-void func_80B37364(EnMaruta* this) {
+void EnMaruta_StartUnderFloor(EnMaruta* this) {
     this->actor.world.pos.y = this->actor.home.pos.y - 630.0f;
-    this->actionFunc = func_80B3738C;
+    this->actionFunc = EnMaruta_RiseThroughFloor;
```

Comment:
We have typically named functions like this in a very rigid fashion; if the goal of the function is to primarily set up an `actionFunc` (like this one), then it's called `Setup[Action]`. So this one would be something like `EnMaruta_SetupRiseThroughFloor`.

This decision was made a long time ago, so I want to ask; is this a pattern that people still want to maintain, or would we prefer more "descriptive" setup function names like this one?